Create Blog Post “q1-2023-vonage-developer-news-recap” (#2421)
Nexmo/deved-platform: content/blog/en/q1-2023-vonage-developer-news-recap.md

---
title: "Q1 2023: Vonage Developer News Recap"
description: Read what we've been up to in the first quarter of 2023. New
  products for campaign management, silent authentication, WhatsApp Product
  Messages, WebRTC audio manager and more!
author: benjamin-aronov
published: true
published_at: 2023-04-05T16:42:19.483Z
updated_at: 2023-04-05T16:42:19.522Z
category: release
tags:
  - proactive-connect
  - silent-auth
  - messages-api
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---
# Imagining New Possibilities

> Each new year is a new opportunity to change ourselves, to grow ourselves, and to imagine new possibilities for ourselves. As the first quarter of 2023 comes to an end, it’s a good time to reflect on the opening chapter of the year.\

Reviewer: ```suggestion
Each new year is a new opportunity to change, grow, and imagine new possibilities for ourselves. As the first quarter of 2023 ends, it’s a good time to reflect on the opening chapter of the year.
```

> \

Reviewer: ```suggestion

```

> In Vonage, we’re proud to look back at Q1 and see we’ve realized quite a few of our resolutions. Whether it’s imagining new products or reimagining products by adding new possibilities, we’ve opened the year by opening doors for our developers with new programmable possibilities. 

Reviewer: I just reduced the number of times that we say possibilities. 

```suggestion
In Vonage, we’re proud to look back at Q1 and see we’ve realized quite a few of our resolutions. Whether imagining new products or reimagining products by adding new opportunities, we’ve opened the year by opening doors for our developers with new programmable possibilities. 
```

## New Products

Creating powerful software is our passion. But powerful software requires an equally enjoyable user experience to truly generate adoption. We’re so excited to debut two new products this quarter that bring user experience to center stage: Proactive Connect and Silent Authentication.

### Getting Started With Proactive Connect

> Does your business need to manage campaigns to users, leads, or other large-scale contact groups? Of course, it does! Now with [Proactive Connect](https://developer.vonage.com/en/blog/getting-started-with-the-campaign-manager-api-and-ui), you can schedule and send personalized messages, across multiple channels, to different large-scale contact groups. But the coolest part is that now you can do it all through a GUI interface built right into your Developer Dashboard!

Reviewer: ```suggestion
Does your business need to manage campaigns for users, leads, or other large-scale contact groups? Of course, it does! Now with [Proactive Connect](https://developer.vonage.com/en/blog/getting-started-with-the-campaign-manager-api-and-ui), you can schedule and send personalized messages, across multiple channels, to different large-scale contact groups. But the coolest part is that you can now do it all through a GUI interface built into your Developer Dashboard!
```

### Silent Authentication

Users expect your software to be safe and secure. But they don’t want to jump through a million hoops like filling out One Time Passwords or installing Authenticator Applications to ensure their identity. Thankfully, there’s a new paradigm in security: [Silent Authentication](https://developer.vonage.com/en/blog/introducing-vonage-silent-authentication).

## New Features

Our goal at Vonage is to empower our developers to delight their end users. Our Messages API and Video API recently debuted some new capabilities that provide even more flexibility and security. Try them out and build something delightful! 

### Using WhatsApp Product Messages With Messages API

WhatsApp is loved by millions of users for rich features. Way beyond simple text messaging, users can send multimedia messages, fun button replys, list messages, locations, and so much more.

> For businesses, one of the most important advances is the launch of [Product Messages](https://developer.vonage.com/en/blog/using-whatsapp-product-messages-with-the-vonage-messages-api). This new message type allows a business to showcase products and services, and customers can browse and add products to a cart without leaving the WhatsApp conversation.

Reviewer: ```suggestion
One of the most significant advances for businesses is the launch of [Product Messages](https://developer.vonage.com/en/blog/using-whatsapp-product-messages-with-the-vonage-messages-api). This new message type allows a company to showcase products and services, and customers can browse and add products to a cart without leaving the WhatsApp conversation.
```

## Enhance your Vonage Video Applications with Audio Connector

Developers usually focus on the video component of video calls. But why? There are so many interesting applications to improve the video call experience when you can access and transmit the audio.\

> \

Reviewer: ```suggestion

```

> Our [Audio Connector](https://developer.vonage.com/en/blog/enhance-your-vonage-video-applications-with-audio-connector) does just that! It lets you send raw audio streams from a Vonage Video session, via your own WebSocket server, to external services for further processing. Some use cases might be transcription, translation, content moderation, and much much more!

Reviewer: ```suggestion
Our [Audio Connector](https://developer.vonage.com/en/blog/enhance-your-vonage-video-applications-with-audio-connector) does just that! It lets you send raw audio streams from a Vonage Video session, via your own WebSocket server, to external services for further processing. Some use cases are transcription, translation, content moderation, and more!
```

### Secure your Inbound Media files with the Vonage Messages API 

> With [WhatsApp Messages](https://developer.vonage.com/en/blog/secure-your-inbound-media-files-with-the-vonage-messages-api), users can send a lot more than just text. But what if those photo or video messages contain some sensitive information? You better be sure they are secure. By clicking one button in your developer dashboard and generating a JWT you can!

Reviewer: Maybe you want to add a conclusion paragraph that tells folks they can join our slack, etc. 
```suggestion
With [WhatsApp Messages](https://developer.vonage.com/en/blog/secure-your-inbound-media-files-with-the-vonage-messages-api), users can send a lot more than just text. But what if those photo or video messages contain some sensitive information? You better be sure they are secure. You can by clicking one button in your developer dashboard and generating a JWT!
```

Millions of users love WhatsApp for its rich features. Way beyond simple text messaging, users can send multimedia messages, fun button replies, list messages, locations, and so much more.

One of the most significant advances for businesses is the launch of [Product Messages](https://developer.vonage.com/en/blog/using-whatsapp-product-messages-with-the-vonage-messages-api). This new message type allows a company to showcase products and services, and customers can browse and add products to a cart without leaving the WhatsApp conversation.

## Enhance your Vonage Video Applications with Audio Connector

Developers usually focus on the video component of video calls. But why? There are many interesting applications to improve the video call experience when you can access and transmit the audio.
Our [Audio Connector](https://developer.vonage.com/en/blog/enhance-your-vonage-video-applications-with-audio-connector) does just that! It lets you send raw audio streams from a Vonage Video session, via your own WebSocket server, to external services for further processing. Some use cases are transcription, translation, content moderation, and more!

> ### Secure your Inbound Media files with the Vonage Messages API 

Reviewer 2: Shouldn't this part be under ##New Features?
It seems out of place here

Reviewer: That sounds like a good idea to me!

Reviewer 3: @DanyF-github what do you mean? It's under New Features
